recipes/fastprng/all/conanfile.py

```diff
@@ -10,23 +14,26 @@ class FastPRNGConan(ConanFile):
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/BrutPitt/fastPRNG"
     settings = "os", "arch", "compiler", "build_type"
-    generators = "cmake",
     no_copy_source = True
 
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
+    def layout(self):
+        basic_layout(self)
 
     def package_id(self):
-        self.info.header_only()
+        self.info.clear()
 
     def validate(self):
-        if self.settings.get_safe("compiler.cppstd"):
-            tools.check_min_cppstd(self, "11")
+        if self.info.settings.get_safe("compiler.cppstd"):
```

Comment:
```suggestion
        if self.settings.get_safe("compiler.cppstd"):
```
Keep `self.settings` when using `self.info.clear()`, because self.info returns `None` always.

Reply:
This way I get `ConanException: 'self.settings' access in 'validate()' method is forbidden`

Reply:
It should be fixed on Beta-5. Remember, Conan 2.0 is still Beta 😉

Reply:
Our focus is making sure we dont break 1.x since the majority still run that. this will be fixed in the 2,0 beta :)

Reply:
Ok, it just feels wrong to send a pull request that does not work locally